engines/sword25/gfx/image/vectorimagerenderer.cpp

```diff
@@ -50,7 +50,7 @@ void art_rgb_fill_run1(byte *buf, byte r, byte g, byte b, int n) {
 		memset(buf, g, n + n + n + n);
 	} else {
 		uint32 *alt = (uint32 *)buf;
-		uint32 color = BS_ARGB(r, g, b, 0xff);
+		uint32 color = TS_RGB(r, g, b);
```

Comment:
Why?

Reply:
The original version only worked because the components were listed in the wrong order (the proper order is a, r, g, b), since the `TransparentSurface` class and the Sword25 engine use different pixel formats internally.